docs/content/concepts/metadata-tags/asset-metadata.mdx

```diff
@@ -307,6 +311,26 @@ Refer to the [Integrating asset metadata into Dagster+ Insights](/dagster-plus/i
 | <PyObject object="MaterializeResult" /> | An object representing a successful materialization of an asset.                                                                         |
 | <PyObject object="MetadataValue" />     | Utility class to wrap metadata values passed into Dagster events, which allows them to be displayed in the Dagster UI and other tooling. |
 
+---
+
+## Reference: standard asset metadata entries
+
+The following is a set of standard asset metadata entries that can be included in the dictionaries that you pass to `metadata` attributes of <PyObject object="asset" decorator />, <PyObject object="MaterializeResult" />, etc. Many of these receive special treatment in Dagster's UI.
+
+The "dagster" prefix indicates that the Dagster package takes responsibility for defining the meaning of these metadata entries.
```

Comment:
```suggestion
The `dagster` prefix indicates that the Dagster package takes responsibility for defining the meaning of these metadata entries.
```

I don't think I've seen us really use `Dagster package` anywhere - usually just `Dagster`. Thoughts on dropping `package`?

Reply:
I think this is an exceptional case where the word "package" is useful because of its precision - i.e. the reason that we're using the string "dagster" inside the metadata key is because it corresponds exactly to the name of the published python package. "Dagster" can have a more broad meaning - encompassing the UI, etc. which live in different packages.